docs/dev-tools/chopsticks/overview.md

```diff
@@ -207,6 +231,160 @@ npx @acala-network/chopsticks run-block  \
 --block 1000
 ```
 
+??? function "**Output file content**"
+  
+    ```json
```

Comment:
This should be in a code snippet in the snippets directory. JSON should be indented 4 spaces https://github.com/papermoonio/documentation-style-guide/blob/main/style-guide.md#code-formatting-by-language

Reply:
This is quite a long snippet, is the entire thing needed?

Reply:
I know it's long, but it displays the entire output of the state changes from one block to another. That's why I tried to use the expandable block to hide it. Maybe we can try to only keep some fields but it wouldn't be the exact output

Reply:
That's fine, we can keep it as is